app/apollo/resolvers/channel.js

```diff
@@ -973,8 +977,10 @@ const channelResolvers = {
           logger.warn({ver_uuid: uuid, ver_name: deployableVersionObj.name}, `${queryName} channel for version ${uuid} not found, authorization assumed.`);
         }
         else {
-          // Channel is found, verify authorization on the Channel
+          // Channel is found, validate if user is authorized for the requested action
+          logger.info({req_id, user, org_id, uuid}, `${queryName} validating`);
```

Comment:
This log line shouldn't be moved from its prior location just after the try{ line -- the lines between there and here are also 'validation'.